.github/workflows/stale.yml

```
@@ -55,6 +58,8 @@ jobs:
           exempt-issue-labels: 'stale::recovered,good-first-issue,help-wanted,severity::1,source::partner,¡important!,¡security!,type::tech-debt,Epic,priority-high'
           # Issues with these labels will never be considered stale
           exempt-pr-labels: 'stale::recovered,good-first-issue,help-wanted,severity::1,source::partner,¡important!,¡security!,type::tech-debt,Epic,priority-high'
+          # The issue labels that have a different stale/close timeline from the rest
```

Let's leave this comment generic:
```suggestion
          # Only issues with these labels are checked whether they are stale
```